### PR TITLE
Add a bunch of stuff.

### PR DESCRIPTION
**Pull request by Mirage-Xel**
-Add the following modules:
Snack Attack
Security Council
Jackbox.TV
Johnson Solids
The Close Button
-Add White Arrows' icon.
-Fix Bigrgunch22 Being credited as CrunchyBot in two modules.

### CODE REVIEW

File: HTML/img/Component/Musical Transposition.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...6h31.796v101.155H98.425zM130.221 123.236h31.325v101.155h-31.325z"/><rect width="30.037" height="101.155" x="161.546" y="123.236" ry="0" fill="#fff" fill-rule="evenodd" stroke="#000" stroke-width="1.293"/><rect width="30.037" height="101.155" x="191.583" y="123.236" ry="0" fill="#fff" fill-rule="evenodd" stroke="#000" stroke-width="1.291"/><rect width="28.75" height="101.155" x="221.62" y="123.236" ry="0" fill="#fff" fill-rule="evenodd" stroke="#000" stroke-width="1.293"/><rect width="22.012" height="58.479" x="49.937" y="123.282" ry="0" fill="#000009" fill-rule="evenodd" stroke="#000"/><path fill="#000009" fill-rule="evenodd" stroke="#000" d="M91.333 123.282h23.326v58.479H91.333z"/><path fill="#000" fill-rule="evenodd" stroke="#000" d="M180.695 123.282h22.012v58.808h-22.012z"/><path fill="#00001e" fill-rule="evenodd" stroke="#000" d="M217.491 123.61h20.698v58.151h-20.698z"/><path fill="#000" fill-rule="evenodd" stroke="#000" d="M142.584 123.282h23.326v58.479h-23.326z"/><text style="line-height:1.25" x="58.295" y="75.805" font-weight="400" font-size="40" font-family="sans-serif" letter-spacing="0" word-spacing="0" fill="#fff"><tspan x="58.295" y="75.805">DISPLAY</tspan></text></svg>
```
Comment (samfundev) [inline HTML/img/Component/Musical Transposition.svg:1]: \<text\> elements should be converted to <path> elements.

File: HTML/img/Component/The Close Button.svg [hunk 2]
```diff
@@ -0,0 +1,234 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (samfundev) [inline HTML/img/Component/The Close Button.svg:1]: Component SVGs should be minified with SVGOMG.

**Review by samfundev — CHANGES_REQUESTED** [review]
I found 2 problems running this PR through the linter.

**Review by MrPeanut1028 — CHANGES_REQUESTED** [review]
There's a ton of merge conflicts as well as a lot of `.meta` files, that'll need to be removed / fixed. Based on the number of fixes that need to be made here, Rdzanu's PR will probably be merged before yours, so it'd be worth making sure you two both didn't make the same changes.

**Pull request closed without merging**